resources/assets/components/utilities/ScholarshipCard/ScholarshipCard.js

```diff
@@ -114,12 +118,16 @@ const ScholarshipCard = ({ campaign }) => {
           </div>
         </div>
 
-        <SecondaryButton
-          className="w-full"
-          href={path}
-          text="Apply Now"
-          onClick={handleScholarshipCardShareClick}
-        />
+        {isAuthenticated ? (
```

Comment:
This is actually exported as a function, so we'll need to invoke it to determine the authenticated status (otherwise it'll always evaluate to 'truthy').
```suggestion
        {isAuthenticated() ? (
```